include/xvega-bindings/xvega_bindings.hpp

```diff
@@ -446,10 +488,21 @@ namespace xv_bindings
 
         void parse_color(const input_it& it)
         {
-            xtl::visit([&](auto&& mark_generic)
```

Comment:
This whole part is kind of ugly... Before I had a very elegant solution that I copied from cppreference examples using universal ref., but idk how to reapply it now. You can see it here https://github.com/jupyter-xeus/xvega-bindings/blob/e130242a654a8f92fd796d999914c526bb51c6a9/include/xvega-bindings/xvega_bindings.hpp#L330-L336 for example
If you have any input on how to make it better, I'd appreciate it. :)
But, not really a must, since Sylvain said he's still thinking about making changes to the API of xvega.
Thanks @JohanMabille !